docs/plugins/graphback-client.md

```diff
@@ -27,6 +27,11 @@ apolloClient.mutate(createUserQuery)
 ```
 
 Plugin allows to create files in many formats to suit the needs of your application.
+The client documents for graphql format are generated in a single file and this should be used by generators, like;
+[GraphQL Code Generator](https://graphql-code-generator.com/)
+[Apollo Generator](https://github.com/apollographql/apollo-tooling#code-generation)
+
+You can also generate the client documents in typescript for direct use if you are not use a generator
```

Comment:
`You can also generate the client documents in typescript for direct use if you are not using a the above mentioned generators`.

Reply:
@wtrocki didn't limit `graphql` format to only the mentioned generators. What if more are created?

Reply:
This is to show people examples. There are couple other generators already but we mentioning 2 most popular ones.

Reply:
No problem.